pkg/apis/openebs.io/v1alpha1/cstorpool_cluster.go

```diff
@@ -35,6 +35,19 @@ const (
 	PoolRaidz2 PoolType = "raidz2"
 )
 
+var (
+	// SupportedPRaidType is a map holding the supported raid configurations
+	// Value of the keys --
+	// 1. In case of striped this is the minimum number of disk required.
+	// 2. In all other cases this is the exact number of disks required.
+	SupportedPRaidType = map[PoolType]int{
+		PoolStriped:  1,
+		PoolMirrored: 2,
+		PoolRaidz:    3,
+		PoolRaidz2:   6,
```

Comment:
If it is so... I will change it.

Reply:
couple of things here:
even though zfs allows of different size raid group, being of different sizes will not help user. So, mostly, they will go with same raid group size. Also, our raid group default size is fixed. This can become configurable.
If same above logic applied for hybrid pool where one raidgroup is mirror and other is stripe, zfs allows it, but, it can as well won't help user.
But, this second case is done by user with explicit mention, which means, he want it. First case can be by mistake as well as he missed the count.
In short, this exact check for raid group size is fine.